dev-packages/e2e-tests/test-applications/create-remix-app-express-vite-dev/app/entry.server.tsx

```diff
@@ -1,25 +1,25 @@
+import * as Sentry from '@sentry/remix';
+
+Sentry.init({
```

Comment:
@mydea, @AbhiPrasad

We should update the wizard and the docs for this right? Also a breaking change warning for the changelog?

Reply:
We can get to the docs/wizard changes afterwards, this is only going to be released with an alpha.

Though we maybe should add a migration note 🤔